### ISSUE TITLE
Accessibility - Add exception for ARIA role="text"

### ISSUE THREAD

**Issue by greenc05** [GoogleChrome/lighthouse#9075]
Invalid ARIA `role="text"` can be used to stop 'text splitting' in VoiceOver iOS https://axesslab.com/text-splitting/ This improves the user experience and does not negatively affect accessibility. 

Please add an exception for Accessibility to Lighthouse, so that the presence of ARIA `role="text"` does not cause the following failure: '[role] values are not valid - ARIA roles must have valid values in order to perform their intended accessibility functions.'

**Comment by patrickhulce** [comment]
Thanks for filing! This logic is handled by the axe library. Would you mind filing this issue [over in their repo](https://github.com/dequelabs/axe-core/issues)?

**Comment by greenc05** [comment]
Thanks @patrickhulce I've just done that

**Comment by brendankenny** [comment]
Fixed in axe-core 4.2.0, which was part of Lighthouse 8
